make.sh

```diff
@@ -98,7 +98,7 @@ if ! [ -f "$key" ] ; then
 fi
 
 
-## Based on https://code.google.com/chrome/extensions/crx.html
+## Based on https://developer.chrome.com/apps/crx
```

Comment:
It seems that Google has modified the site not to includes the sample packaging script here...

Reply:
The original URL https://code.google.com/chrome/extensions/crx.html redirects to https://developer.chrome.com/extensions/hosting , but the new URL https://developer.chrome.com/apps/crx redirects to https://developer.chrome.com/extensions/linux_hosting . I'm not clear what you intended, but it should probably be the final URL, after redirects.